tests/unit/stream_alert_athena_partition_refresh/test_main.py

```diff
@@ -89,15 +89,10 @@
             ]
         },
         'athena_partition_refresh_config': {
-            "enabled": True,
-            "refresh_type": {
-                "repair_hive_table": {
-                    "unit-testing.streamalerts": "alerts"
-                },
-                "add_hive_partition": {
-                    "unit-testing.streamalerts": "alerts",
-                    "unit-testing.streamalert.data": "data"
-                }
+            'enabled': True,
```

Comment:
This is no longer used in the config, let's also remove it from the test

Reply:
yes good catch - I'll remove :)